packages/core/test/financial-templates/contract-for-difference/ContractForDifference.js

```diff
@@ -96,6 +95,7 @@ contract("ContractForDifference", function (accounts) {
       collateralTokenAddress: collateralToken.address,
       finderAddress: finder.address,
       contractForDifferenceLibraryAddress: contractForDifferenceLibrary.address,
+      ancillaryData,
```

Comment:
cool but should you not add a test also that this `ancillaryData` is included within the OO price requests?

Reply:
I think this is fixed? I added a test case where I grab the request from the OO and use the ancillary data to look it up. Is that what you were looking for? PTAL